src/lib/Debridge/index.ts

```diff
@@ -0,0 +1,30 @@
+import { protocols } from '../../config';
+import { ProtocolHelper } from '../../helpers';
+import { IProtocolContractDefinitions, IProtocolParserExport, ITransaction, ITransactionAction } from '../../types';
+import { CONTRACT_ENUM, contracts } from './contracts';
+import { DlnSourceContractParseTransaction, DlnDestinationContractParseTransaction } from './parser';
+
+export default class Debridge implements IProtocolParserExport {
+  public readonly protocolIdentifier: string;
+
+  constructor() {
+    this.protocolIdentifier = protocols.debridge.identifier;
```

Comment:
Please define debridge under protocols and add the necessary info

Reply:
Done